### ISSUE TITLE
The method 'returnHue' was called on null

### ISSUE THREAD

**Issue by mishatron** [knezzz/RandomColor#9]
I am using Flutter 1.20.0 • channel beta and got error: The method 'returnHue' was called on null

**Comment by knezzz** [comment]
Hey @mishatron,

What did you call when this happened? Could you provide me with some code that will reproduce this issue?

**Comment by mishatron** [comment]
@knezzz I used method to get random color from your library. And it worked earlier, maybe the problem is in flutter sdk changes. As I understand ColorHue colorHue = ColorHue.random, dont work if you pass null. It only works if you dont pass any parameter. What I suggest to do - is to add 3 lines of code to the method:
```
Color randomColor({
    ColorHue colorHue = ColorHue.random,
    ColorSaturation colorSaturation = ColorSaturation.random,
    ColorBrightness colorBrightness = ColorBrightness.random,
    bool debug = false,
  }) {
    this.debug = debug;

    int s;
    int h;
    int b;

    colorHue ??= ColorHue.random;//new line
    colorSaturation ??= ColorSaturation.random;//new line
    colorBrightness ??= ColorBrightness.random;//new line
    h = colorHue.returnHue(_random);
    s = colorSaturation.returnSaturation(_random);
    b = colorBrightness.returnBrightness(_random);

    _log('Color hue: $h');
    _log('Color saturation: $s');
    _log('Color brightness: $b');

    return _getColor(h, s, b);
  }
```

**Comment by lukaknezic** [comment]
Yeah okay, makes sense, I will update that on `randomColor` and `materialColor` getters.

**Comment by knezzz** [comment]
Yeah makes sense, I will add that on `randomColor` and `materialColor` getters

**Comment by knezzz** [comment]
Published in `1.0.6-nullsafety`
